packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.scss

```diff
@@ -32,6 +32,23 @@
   }
 }
 
+/*
+ * When the DetailsList hover mode is "above" and the list is in hover state,
+ * make all checkboxes visible by default and hide checkboxes in rows below the hovered row.
+ */
+:global(.ms-DetailsList--HoverAbove:hover) {
```

Comment:
is this level of specificity really necessary? are there other checks not inside .owner?

Reply:
afaict, we only ever render DetailsRowChecks within containers which have the `.owner` CSS class applied. Doesn't look like the specificity is needed; I've removed it now 👍